server/src/main/java/org/elasticsearch/action/search/SearchRequest.java

```diff
@@ -253,7 +253,8 @@ public SearchRequest(StreamInput in) throws IOException {
             finalReduce = true;
         }
         ccsMinimizeRoundtrips = in.readBoolean();
-        if (in.getTransportVersion().before(TransportVersions.RE_REMOVE_MIN_COMPATIBLE_SHARD_NODE) && in.readBoolean()) {
+        if ((in.getTransportVersion().isPatchFrom(TransportVersions.RE_REMOVE_MIN_COMPATIBLE_SHARD_NODE_90)
```

Comment:
I think this has to be == false?
They are using "before", so it means that this branch is probably for the "old" path?

Reply:
Thanks 👍  totally missed this is using `before` rather than `after`